pay-api/src/pay_api/services/statement.py

```diff
@@ -138,12 +144,67 @@ def find_by_account_id(auth_account_id: str, page: int, limit: int):
         return data
 
     @staticmethod
-    def get_statement_report(statement_id: str, content_type: str, template_name='statement_report', **kwargs):
+    def get_statement_template(statement: StatementModel, statement_invoices: List[InvoiceModel]) -> str:
```

Comment:
suggestion: `ordered_statement_invoices` or `ordered_invoices`

Reply:
Updated to ordered_invoices